core/renderers/thrasos/info.js

```diff
@@ -11,8 +11,8 @@
  */
 'use strict';
 
-goog.provide('Blockly.thrasos');
```

Comment:
Just out of curiosity: what's with this?  Any idea why this file was providing `Blockly.thrasos` when it didn't actually export any properties other than `Blockly.thrasos.RenderInfo`?

Reply:
I talked with Rachel and she said that it was fine to remove.
It's possible it was added because it was thought that there needed to be a provide for the intermediate namespace.
It's also not needed as an entry point for getting all the requires for the renderer because we use `Blockly.thrasos.Renderer` for that.